internal/confmapprovider/discovery/bundle/bundle.d/receivers/mysql.discovery.yaml.tmpl

```diff
@@ -0,0 +1,60 @@
+{{ receiver "mysql" }}:
+  enabled: true
+  rule:
+    docker_observer: type == "container" and any([name, image, command], {# matches "(?i)mysql"}) and not (command matches "splunk.discovery")
+    host_observer: type == "hostport" and command matches "(?i)mysqld"
```

Comment:
When I test this I see that the x protocol port will also be included by this rule (also affects collectd monitor). We may want to tune it to include something like `and port % 10 != 0` since the default port is 33060.

```
scrapercontroller.go:200        Error scraping metrics        {"kind": "receiver", "name": "receiver_creator/discovery", "data_type": "metrics", "name": "mysql//receiver_creator/discovery{endpoint=\"127.0.0.1:33060\"}/(host_observer)127.0.0.1-33060-TCP-4040", "error": "MySQL server does not support required protocol 41+; MySQL server does not support required protocol 41+; MySQL server does not support required protocol 41+; MySQL server does not support required protocol 41+; MySQL server does not support required protocol 41+", "scraper": "mysql"}
```